bin/deploy

```diff
@@ -1,24 +1,17 @@
 #/bin/sh
 
-# Set Symfony environment to drupal_prod
-export SYMFONY_ENV=drupal_prod
-
 # Disable Symfony debug mode
 export SYMFONY_DEBUG=0
```

Comment:
This is also not needed.

Reply:
The export above does not do anything in Symfony 4. Unless it's used for something else, it can be removed.